rethinkdb/content.md

```diff
@@ -0,0 +1,50 @@
+# What is RethinkDB?
+
+RethinkDB is an open-source, distributed database built to store JSON documents
+and effortlessly scale to multiple machines. It's easy to set up and learn and
+features a simple but powerful query language that supports table joins,
+groupings, aggregations, and functions.
+
+%%LOGO%%
+
+# How to use this image
+
+## start an instance with data mounted in the working directory
+
+The default CMD of the image is `rethinkdb --bind all`, so the RethinkDB daemon
+will bind to all network interfaces available to the container (by default,
+RethinkDB only accepts connections from `localhost`).
+
+```bash
+docker run --name some-rethink -v "$PWD:/data" -d rethinkdb
+```
+
+## connect to it from an application ...
+
+```bash
+docker run --name some-app --link some-rethink:rdb -d application-that-uses-rdb
+```
+
+## ... via the web admin interface ...
+
+```bash
+# browsing on the same machine:
+xdg-open "http://$(docker inspect --format \
+  '{{ .NetworkSettings.IPAddress }}' some-rethink):8080"
+
+# exposing and connecting remotely:
+docker run -d --link some-rethink:rdb -p 8080:8080 svendowideit/ambassador
```

Comment:
So that'd be, what, `ssh -nNTL 8080:$(ssh user@remote docker inspect --format \
+  '{{ .NetworkSettings.IPAddress }}' some-rethink):8080 user@remote; xdg-open http://localhost:8080`?